airbyte-integrations/connectors/source-square/integration_tests/configured_catalog.json

```diff
@@ -148,13 +148,13 @@
       "stream": {
         "name": "orders",
         "json_schema": {},
-        "supported_sync_modes": ["full_refresh"],
+        "supported_sync_modes": ["incremental","full_refresh"],
         "source_defined_cursor": true,
-        "default_cursor_field": ["id"]
+        "default_cursor_field": ["created_at"]
       },
-      "sync_mode": "full_refresh",
-      "cursor_field": ["id"],
-      "destination_sync_mode": "overwrite"
+      "sync_mode": "incremental",
+      "cursor_field": ["created_at"],
+      "destination_sync_mode": "append"
```

Comment:
@brijeshphilz not sure if this supports append, as we should be using `updated_at` and this would be an overwrite?

We can also define `primary_key` for overwriting i.e. `"primary_key": [["id"]]`